packages/patternfly-4/react-core/src/experimental/components/Drawer/Drawer.tsx

```diff
@@ -11,24 +11,19 @@ export interface DrawerProps extends React.HTMLProps<HTMLDivElement> {
   isExpanded?: boolean;
   /** Indicates if the content element and panel element are displayed side by side. */
   isInline?: boolean;
-  /** Content rendered in the drawer */
-  panelContent?: React.ReactNode;
```

Comment:
Experimental :tada:

Reply:
Exactly! Good way we added it in experimental so we can improve the API before it's full fledged component.